src/components/PageLayout/SimpleLoader.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <span class="loader"></span>
```

Comment:
This should probably have an `aria-label` for accessibility

Reply:
Good catch, I have added it.